### PR TITLE
move solutions list to contentful

### PR DESCRIPTION
**Pull request by OleksandrDragun**
### PR Checklist

* [ ] Have you followed the guidelines in our [Dev Guide](../#readme)
* [ ] Have you lint your code locally prior to submission (`npm run lint`)?
* [ ] Have you run prettier (`npm run format`)?
* [ ] Have you checked that app can be built (`npm run build`)?
* [ ] Have you checked that the build output looks according to design (screen, mobile, desktop) in Figma (`npm run serve`)?

### CODE REVIEW

File: src/components/Layout/Navigation/NavMenu/LinkList/LinkList.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import React, { FC } from 'react';
+import chunk from 'lodash/chunk';
+import { createBemBlockBuilder } from '@app/utils';
+
+import { LinkListItem } from './LinkListItem';
+
+import './LinkList.scss';
+
+export type ListItem = {
```
Comment (ViktorSoroka07) [inline src/components/Layout/Navigation/NavMenu/LinkList/LinkList.tsx]: Let's have it as interface for consistency

Comment (OleksandrDragun) [reply src/components/Layout/Navigation/NavMenu/LinkList/LinkList.tsx]: No problem. Done.

File: src/components/Layout/Navigation/NavMenu/LinkList/LinkList.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+}
+
+export const LinkList: FC<LinkListProps> = ({
+  title,
+  items,
+  showTitle = true,
+  itemsPerRow = items.length,
+  className = '',
+}) => {
+  const getBlocksWith = createBemBlockBuilder(['link-list', className as string]);
```
Comment (ViktorSoroka07) [inline src/components/Layout/Navigation/NavMenu/LinkList/LinkList.tsx]:
```suggestion
  const getBlocksWith = createBemBlockBuilder(['link-list', className]);
```

File: src/components/Layout/Navigation/NavMenu/LinkList/LinkListItem/LinkListItem.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+export const LinkListItem: FC<LinkListItemProps> = ({
+  title,
+  link,
+  icon,
+  description,
+  className = '',
+}) => {
+  const getBlocksWith = createBemBlockBuilder(['link-list-item', className as string]);
```
Comment (ViktorSoroka07) [inline src/components/Layout/Navigation/NavMenu/LinkList/LinkListItem/LinkListItem.tsx]:
```suggestion
  const getBlocksWith = createBemBlockBuilder(['link-list-item', className]);
```

**Review by ViktorSoroka07 — DISMISSED** [review]

**Review by AmsterGet — DISMISSED** [review]

File: src/components/Layout/Navigation/NavMenu/LinkList/LinkList.tsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+import './LinkList.scss';
+
+export interface ListItem {
+  title: string;
+  description: string;
+  link: string;
+  icon: {
+    url: string;
+  };
```
Comment (ViktorSoroka07) [inline src/components/Layout/Navigation/NavMenu/LinkList/LinkList.tsx]:
```suggestion
  icon: ContentfulAsset;
```

File: src/components/Layout/Navigation/NavMenu/LinkList/LinkListItem/LinkListItem.tsx [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+import { Link } from '@app/components';
+import { createBemBlockBuilder } from '@app/utils';
+
+interface LinkListItemProps {
+  title: string;
+  description: string;
+  link: string;
+  icon: {
+    url: string;
+  };
```
Comment (ViktorSoroka07) [inline src/components/Layout/Navigation/NavMenu/LinkList/LinkListItem/LinkListItem.tsx]:
```suggestion
  icon: ContentfulAsset;
```

File: src/components/Layout/Navigation/NavMenu/LinkList/LinkList.tsx (hunk 2 above)
Comment (OleksandrDragun) [reply src/components/Layout/Navigation/NavMenu/LinkList/LinkList.tsx]: Done.

File: src/components/Layout/Navigation/NavMenu/LinkList/LinkListItem/LinkListItem.tsx (hunk 3 above)
Comment (OleksandrDragun) [reply src/components/Layout/Navigation/NavMenu/LinkList/LinkListItem/LinkListItem.tsx]: Done.

File: src/components/Layout/Navigation/NavMenu/LinkList/LinkList.tsx (hunk 4 above)
Comment (OleksandrDragun) [reply src/components/Layout/Navigation/NavMenu/LinkList/LinkList.tsx]: Done.

File: src/components/Layout/Navigation/NavMenu/LinkList/LinkListItem/LinkListItem.tsx (hunk 5 above)
Comment (OleksandrDragun) [reply src/components/Layout/Navigation/NavMenu/LinkList/LinkListItem/LinkListItem.tsx]: Done.

**Review by ViktorSoroka07 — APPROVED** [review]

**Pull request merged**